src/DotNet.Status.Web/Startup.cs

```diff
@@ -268,7 +268,7 @@ private void AddServices(IServiceCollection services)
                             policy.RequireAuthenticatedUser();
                             if (!Env.IsDevelopment())
                             {
-                                policy.RequireRole(GitHubClaimResolver.GetTeamRole("dotnet","dnceng"), GitHubClaimResolver.GetTeamRole("dotnet","bots-high"));
+                                policy.RequireRole(GitHubClaimResolver.GetTeamRole("dotnet","dnceng-public"), GitHubClaimResolver.GetTeamRole("dotnet","bots-high"));
```

Comment:
This shouldn't be changed. This is a reference to a github team, not the azdo org.